app/controllers/api/v1/games_controller.rb

```diff
@@ -1,16 +1,26 @@
 module Api::V1
   class GamesController < ApplicationController
 
+    before_action :conflict_error, if: -> { lobby.game.present? },
```

Comment:
What about a 422 Unprocessable Entity? [This blog post](https://localize-software.phraseapp.com/posts/best-practice-10-design-tips-for-apis/) suggests that status code for validation errors (which this is similar to).